wordpress-mobile/WordPress-iOS
# Remove ReaderPost content format from the Service

Fixes #14313

`WordPressKit` related PR: https://github.com/wordpress-mobile/WordPressKit-iOS/pull/251

This PR removes the `RichContentFormatter` from the Reader's service and move it to `ReaderDetailViewController`.

This change is necessary because the formatting is stripping some HTML tags that are making the post to render incorrectly in the new ReaderDetail that uses a WebView.

We lose a little bit of performance for doing that way, but soon the rendering with a WebView will be available and that will not be an issue.

### To test

1. Open posts in the Reader
2. Make sure they appear the same way as it was before
